docs/source/multi_gpu.rst

```diff
@@ -654,7 +654,7 @@ To use Sharded Training, you need to first install FairScale using the command b
 
 .. code-block:: bash
 
-    pip install https://github.com/PyTorchLightning/fairscale/archive/pl_1.1.0.zip
+    pip install fairscale
```

Comment:
the changes to this file should not be here

Reply:
looks like rebase wasn't done correctly.